src/actions/ui.js

```diff
@@ -37,3 +37,8 @@ export const userDismissedNotification = createAction(
   'USER_DISMISSED_NOTIFICATION',
   type => ({type}),
 );
+
+export const setRefreshTimestamp = createAction(
+  'SET_REFRESH_TIMESTAMP',
```

Comment:
Very low-level comment but as a naming convention I generally like to try to describe user-initiated actions from the standpoint of user intent, so this one something like `REFRESH_PREVIEW`.